config/locales/simple_form.en.yml

```diff
@@ -13,7 +13,7 @@ en:
           url: Fedora URL should not end with a slash
         name: A single or hyphenated name used for technical aspects of the repository (e.g., "acme" or "acme-library").
         gtm_id: The ID of your Google Tag Manager account
-        contact_email: Email recipient of messages sent via the contact form
+        contact_email: The email address that messages submitted via the contact page are sent to
```

Comment:
the contact_email and contact_email_to descriptions are swapped. Apologies for merging in that fix before merging this PR in.